### PR TITLE
New dev sandbox dk

### PR DESCRIPTION
**Pull request by rachidatecs**
## Ticket

Part of #900 

## Changes

- Create manifest for dk
- Add dk to workflows
- Update Org name in ops and workflow files 
- Add dk to ALLOWED_HOSTS

## Context for reviewers

Not sure whether the Org name update is needed, not needed bu benign, or inadvisable at this point.

### CODE REVIEW

File: src/registrar/tests/test_views.py [hunk 1]
```diff
@@ -1500,3 +1500,19 @@ def test_application_status_no_permissions(self):
                         reverse(url_name, kwargs={"pk": application.pk})
                     )
                     self.assertEqual(page.status_code, 403)
+
+
+def test_approved_application_not_in_active_requests(self):
```
Comment (neilmb) [inline src/registrar/tests/test_views.py:1505]: **comment**: It looks like this probably crept in from a separate workstream. Is it easy enough leave this change out of this PR?

**Review by neilmb — COMMENTED** [review]
Thanks for setting this up Rachid. This PR will also make `cisa-dotgov` our default organization for sandboxes, etc. which is awesome!

File: .github/workflows/reset-db.yaml [hunk 2]
```diff
@@ -25,6 +25,7 @@ on:
           - ab
           - bl
           - rjm
+          - dk
```
Comment (abroddrick) [inline .github/workflows/reset-db.yaml:28]: did you manually add this? the script should auto add this to the top under staging

**Review by abroddrick — APPROVED** [review]
Thanks!

File: .github/workflows/reset-db.yaml (hunk 2 above)
Comment (rachidatecs) [reply .github/workflows/reset-db.yaml:28]: Yes @abroddrick , I manually added most of these. The scripts had a stroke at the build step, due to either me having docker composed up already or being in the wrong directory. I had to run the rest of the steps manually and then edit the files by hand.

**Review by therealslimhsiehdy — APPROVED** [review]
Thank you!! 👯

**Pull request merged**